docs/source/developer_guide/continuous_integration.rst

```diff
@@ -13,13 +13,22 @@ standards such as a common `coding style <#Coding-style>`__.
 Code quality checks
 -------------------
 
-.. _precommit: https://pre-commit.com
+.. _pre-commit: https://pre-commit.com
 
-We use `pre-commit <precommit>`_ for code quality checks.
-These checks run automatically before you make a new commit.
+We use `pre-commit`_ for code quality checks (a process we also refer to as "linting" checks).  We recommend that you set this up locally as it will
+insure that you never run into code-quality errors when you make your first PR! These checks run automatically before you make a new commit.
+To setup, first install precommit in your developer environment:
+
+::
+   pip install pre-commit
+
+Then navigate the the sktime folder and install our pre-commit configuration:
+
+::
+   pre-commit install
```

Comment:
that's a good spot - did we not have this in the guidelines?

Reply:
At least not in the path that I took